docs/data/advanced-components/overview.md

```diff
@@ -0,0 +1,172 @@
+---
+title: Advanced Components - Overview
+---
+
+# Advanced Components - Overview
+
+<p class="description">A collection of fast and powerful components.</p>
+
+## Licenses
+
+While MUI Core is entirely licensed under MIT, MUI X serves a part of its components under a commercial license.
+Please pay attention to the license.
+
+### Community Plan
+
+Published under [MIT license](https://tldrlegal.com/license/mit-license) and [free forever](https://mui-org.notion.site/Stewardship-542a2226043d4f4a96dfb429d16cf)
+
+The packages of the **Community** Plan are:
+
+- [`@mui/x-data-grid`](https://www.npmjs.com/package/@mui/x-data-grid)
+- [`@mui/x-date-pickers`](https://www.npmjs.com/package/@mui/x-date-pickers)
+
+### Pro Plan
+
+This plan contains the features that are at the limit of what the open-source model can sustain. From a price perspective, the plan is designed to be accessible to most professionals.
+
+See [Pricing](https://mui.com/store/items/material-ui-pro/) for details on purchasing licenses.
+
+The packages of the **Pro** Plan are:
+
+- [`@mui/x-data-grid-pro`](https://www.npmjs.com/package/@mui/x-data-grid-pro)
+- [`@mui/x-date-pickers-pro`](https://www.npmjs.com/package/@mui/x-date-pickers-pro)
+
+The features exclusives to the **Pro** Plan are marked with the <span class="plan-pro"></span> icon across our documentation.
+
+<div class="only-light-mode">
+  <img src="/static/x/commercial-header-icon-light.png" style="width: 579px; margin-bottom: 2rem;" alt="">
+</div>
+<div class="only-dark-mode">
+  <img src="/static/x/commercial-header-icon-dark.png" style="width: 560px; margin-bottom: 2rem;" alt="">
+</div>
+
+### 🚧 Premium Plan
+
+This plan contains the most advanced features.
+
+The features exclusives to the **Premium** Plan are marked with the <span class="plan-premium"></span> icon across our documentation.
+The **Premium** Plan also contains all the features of the **Pro** Plan
+
+## MIT vs. commercial
```

Comment:
Pure Copy Paste

Reply:
Copy / pasted
But I agree that we talk a lot about the "why", which was maybe useful one year ago, but now our product feature should justify the cost by themselves.